src/app/models/welding_point.py

```diff
@@ -5,17 +5,20 @@
 
 class WeldingPoint(Base):
     id = Column(Integer, primary_key=True, index=True)
-    project_id = Column(Integer, ForeignKey("project.id"))
-    robot_id = Column(Integer, ForeignKey("robot.id"))
-    welding_order = Column(Integer, index=True)
-    name = Column(String)
+    project_id = Column(Integer, ForeignKey("project.id"), nullable=False)
+    robot_id = Column(Integer, ForeignKey("robot.id"), nullable=False)
+    welding_order = Column(Integer, index=True, nullable=False)
+    name = Column(String, nullable=False)
     description = Column(String)
-    x = Column(Float)
-    y = Column(Float)
-    z = Column(Float)
-    roll = Column(Float)
-    pitch = Column(Float)
-    yaw = Column(Float)
+    x = Column(Float, nullable=False)
+    y = Column(Float, nullable=False)
+    z = Column(Float, nullable=False)
+    x_original = Column(Float, nullable=False)
+    y_original = Column(Float, nullable=False)
+    z_original = Column(Float, nullable=False)
+    roll = Column(Float, nullable=False)
+    pitch = Column(Float, nullable=False)
+    yaw = Column(Float, nullable=False)
```

Comment:
Doing roll, pitch and yaw not nullable depends on how we want to handle those values later. At a certain point, we won't get these values from the Excel files anymore as we only get the normal vector. Do we calculate those values when we add welding points via endpoint and add it to the object directly? Or do we just calculate it right before the codegen? For the second case, these values can be nullable.

Was not sure what our goto is now.

Reply:
Right I think we should keep them mandatory.
But actually, these will disappear once #55 is implemented.

At that point I think the `norm_vector_x/y/z` should also be mandatory even if we just assign a default value because if I remember correctly niryo_one_ros doesn't have roll/pitch/yaw optional either and a robot **has** to have roll/pitch/yaw given to know how to move.